presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -443,11 +443,11 @@ public void testStructTimestamps(StorageFormat format)
 
         onPresto().executeQuery(format(
                 "CREATE TABLE %s (\n"
-                        + "id INTEGER\n"
-                        + ",arr ARRAY(TIMESTAMP)\n"
-                        + ",map MAP(TIMESTAMP, TIMESTAMP)\n"
-                        + ",row ROW(col TIMESTAMP)\n"
-                        + ",nested ARRAY(MAP(TIMESTAMP, ROW(col ARRAY(TIMESTAMP))))\n"
+                        + "id INTEGER,\n"
+                        + "arr ARRAY(TIMESTAMP),\n"
+                        + "map MAP(TIMESTAMP, TIMESTAMP),\n"
+                        + "row ROW(col TIMESTAMP),\n"
+                        + "nested ARRAY(MAP(TIMESTAMP, ROW(col ARRAY(TIMESTAMP))))\n"
```

Comment:
I prefer the SQL comma style from the earlier commits because it makes it easier to avoid missing commas. But it also looks weird, so here's a commit I can squash if other people don't like it.

(The `\n`s are there for similar reasons, to make it easier to see when there is or isn't whitespace between the lines.)